here/traffic/here_api.py

```diff
@@ -108,6 +122,7 @@ def get_download_url(request_id, status_base_url, access_token, user_id):
         LOGGER.info('Polling status of query request: %s', request_id)
         query_status = requests.get(status_url, headers = status_header)
         try:
+            query_status.raise_for_status()
```

Comment:
Should probably include the same `except requests.exceptions.HTTPError as err:` from above